src/WebJobs.Script.WebHost/Diagnostics/WebHostMetricsLogger.cs

```diff
@@ -15,6 +15,7 @@ public void BeginEvent(MetricEvent metricEvent)
             if (startedEvent != null)
```

Comment:
Following kudu's examples, I was able to replace EventProvider with EventSource. Now, the project doesn't use unsafe and unchecked segments anymore.